frappe/core/doctype/doctype/doctype.py

```diff
@@ -396,10 +396,7 @@ def after_rename(self, old, new, merge=False):
 			frappe.db.sql("""update tabSingles set value=%s
 				where doctype=%s and field='name' and value = %s""", (new, new, old))
 		else:
-			frappe.db.multisql({
-				"mariadb": f"RENAME TABLE `tab{old}` TO `tab{new}`",
-				"postgres": f"ALTER TABLE `tab{old}` RENAME TO `tab{new}`"
-			})
+			frappe.db.sql(frappe.qb.rename_table(old,new))
```

Comment:
This should be a part of the ORM. Under `frappe.db.rename` or `frappe.db.rename_table`. Logic can be moved to https://github.com/frappe/frappe/blob/develop/frappe/database/postgres/database.py and https://github.com/frappe/frappe/blob/develop/frappe/database/mariadb/database.py respectively.


Similarly for `frappe.qb.DESC`